README.md

```diff
@@ -99,24 +99,41 @@ on:
 
 jobs:
   cache-and-install:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
 
     steps:
-      build:
-        - uses: actions/checkout@v2
-
-        - name: Cache pnpm modules
-          uses: actions/cache@v2
-          with:
-            path: ~/.pnpm-store
-            key: ${{ runner.os }}-${{ hashFiles('**/pnpm-lock.yaml') }}
-            restore-keys: |
-              ${{ runner.os }}-
-
-        - uses: pnpm/action-setup@v2.1.0
-          with:
-            version: 6.0.2
-            run_install: true
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
Other examples don't have this. Is this necessary for cache? If not, then I think the example should be simplified.

```suggestion
```

Reply:
Oh. No, actually. That is specific to keep history for other steps I used in my based project.

I'll remove shortly.

Reply:
you just need to use the "commit suggestion" button in the github interface, lol.

Reply:
Right! 😅 I just wanted to keep 1 simple commit.